compiler/rustc_session/src/config.rs

```diff
@@ -1400,7 +1400,9 @@ The default is {DEFAULT_EDITION} and the latest stable edition is {LATEST_STABLE
 pub fn rustc_short_optgroups() -> Vec<RustcOptGroup> {
     vec![
         opt::flag_s("h", "help", "Display this message"),
-        opt::multi_s("", "cfg", "Configure the compilation environment", "SPEC"),
+        opt::multi_s("", "cfg", "Specify the features defined in the manifest
+                             and compile the relevant components based on
+                             cfg attributes.", "SPEC"),
```

Comment:
This is describing the purpose of the option, but not how to actually use it. I think something like this would be more helpful:
"Configure the compilation environment. \
SPEC has the syntax `<name>[=\"value\"]`."
```
I am not sure that is actually the right syntax, you should test it locally to verify (and maybe also read the code to see if there are other syntax options I missed).

Reply:
oh, I see what you mean. I will add some concrete approachs later.